### ISSUE TITLE
Automatic email to crds_datamng

### ISSUE THREAD

**Issue by jwhite3** [Vb2341/ReDCaT#21]
In a meeting with Matt, we figured out a way to make this email automatic and we can tag it to the end of "move_files"

essentially what Matt explained is that when you start a submission to crds, a ".cat" file is created with the list of files that are being submitted. All we'd need to do is read this ".cat" file for the email message. The only tricky bit would be learning how to send emails via python

**Comment by gkanarek** [comment]
[This seems pretty straightforward.](https://docs.python.org/3.5/library/email-examples.html)

```python
import smtplib
from getpass import getuser
from email.mime.text import MIMEText

with open("<submission>.cat") as f:
    delivered_files = f.read()

msg_text = "The following files have been delivered to DMS:\n\n" + delivered_files + \
    "\nRegards, the ReDCaT Team"
msg = MIMEText(msg_text)

msg['Subject'] = "New delivery to DMS"
msg['From'] = getuser()+"@stsci.edu"
msg['To'] = "crds_datamng@stsci.edu"

s = smtplib.SMTP("<whatever_our_email_server_is>")
s.send_message(msg)
s.quit()
```

**Comment by jwhite3** [comment]
Exactly! That's pretty much exactly what I'm doing for the instrument team tool. 

I think the server we use is 'smtp.stsci.edu'

**Comment by gkanarek** [comment]
Any idea where the .cat file is created? There's a "rename.log" in the delivery directory which I could parse, but I'm not sure where else to look.

**Comment by jwhite3** [comment]
I had a though: 

Instead of making this email automatic after a delivery is confirmed, why not just create tools that make the email and send it for the team members? This way they wouldn't be interrupted to craft the messages to crds_datamng
